src/lib/plot/xy-plot.js

```diff
@@ -225,11 +216,45 @@ class XYPlot extends React.Component {
     };
   }
 
-  render() {
-    const {width, height, animation} = this.props;
+  /**
+   * Checks if the plot is empty or not.
+   * Currently checks the data only.
+   * @returns {boolean} True for empty.
+   * @private
+   */
+  _isPlotEmpty() {
+    const {data} = this.state;
+    return !data || !data.length || ![].concat(...data).filter(d => d).length
```

Comment:
@chrisirhc , `[].concat(...data)` is needed: it's an array of arrays (e.g. array of data for each series) and I have to know if *any* data is passed to it.